Update Python version requirement and bump pyjq to 2.6.0

amaanx86 opened the pull request:
This pull request includes updates to the `README.md` file to reflect the use of Python 3.10 for setting up the virtual environment instead of Python 3.7.

Key changes:

* Updated the Python version requirement from 3.7.0rc1 to 3.10 in the installation requirements section.
* Changed the command to create a virtual environment to use Python 3.10 in the macOS setup instructions.
* Updated the command to create a virtual environment to use Python 3.10 in the Ubuntu setup instructions.…d requirements

amaanx86 committed:
Update Python version requirement and bump pyjq to 2.6.0 in README and requirements

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/duo-labs/cloudmapper?pullRequest=968) <br/>All committers have signed the CLA.

amaanx86 commented:
This pull request updates the installation instructions in the `README.md` to use Python 3.10 instead of older Python 3 versions. This ensures users set up the environment with a compatible and more recent Python release.

Installation requirements update:

* Updated the listed Python requirement from "python 3 (3.7.0rc1 is known to work)" to "python 3 (3.10 is known to work)" in the requirements section.

Setup commands update:

* Changed the virtual environment creation command on macOS and Linux to use `python3.10` instead of `python3` for consistency with the new requirement. [[1]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L63-R63) [[2]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L77-R77)

amaanx86 commented:
Project seems dead - closing it to clear personal backlog
